pysisyphus/calculators/Dimer.py

```diff
@@ -352,13 +353,31 @@ def set_N_raw(self, coords):
             bond_modes = [self.get_bond_mode(bond, coords)
                           for bond in self.bonds]
             N_raw = np.sum(bond_modes, axis=0)
-        # Normalize N_raw
+        # Make N_raw translationally invariant and normalize
         self.N = N_raw
         # Now we keep the normalized dimer orientation
         self.N_raw = self.N
 
         self.log(f"Initial orientation:\n\t{self.N}")
 
+    def make_translationally_invariant(self, coords):
```

Comment:
I would call the method something like `remove_translation()` and rename the `coords` argument to something more general like `arr` or `vec` or even `displacement`.